README.md

```diff
@@ -79,25 +79,27 @@ Finally, enable all of the rules that you would like to use:
 
 # List of supported rules
 
-* [enact/kind-name](docs/rules/display-name.md): Prevent missing `name` in a Enact component definition
-* enact/display-name: Prevent missing `displayName` in a React component definitions without false-flagging Enact kinds
+* [enact/kind-name](docs/rules/kind-name.md): Prevent missing `name` in a Enact component definition
+* [enact/display-name](docs/rules/display-name.md): Prevent missing `displayName` in a React component definitions without false-flagging Enact kinds
 * [enact/prop-types](docs/rules/prop-types.md): Prevent missing props validation in a React component definition
 * [enact/no-module-exports-import](docs/rules/no-module-exports-import.md): Disallow module.exports with import statements (see [webpack issue #4039](https://github.com/webpack/webpack/issues/4039))
 
+> Note: `enact/display-name` and `enact/prop-types` supersede `react/display-name` and `react/prop-types`, respectively. The latter two should not be disabled when used with Enact.
```

Comment:
Should this read "should be disabled"? If not, I'm a bit confused by the wording.

Reply:
Um, yeah, it should.   🤦‍♂